### PR TITLE
Add 'uses_only_baseline_capabilities' field to API v3

### PR DESCRIPTION
**Pull request by leplatrem**
Ref #2047

### CODE REVIEW

**Comment by leplatrem** [comment]
(I hadn't ran the tests locally :grin:, will work on this ;))

File: normandy/recipes/tests/api/v3/test_api.py [hunk 1]
```diff
@@ -144,6 +144,22 @@ def test_list_sets_no_cookies(self, api_client):
...
+            recipe2 = RecipeFactory(extra_capabilities=["test-capability"])
+            assert not recipe2.uses_only_baseline_capabilities()
+
+            res = api_client.get("/api/v3/recipe/")
+            assert res.status_code == 200
+            assert res.data["count"] == 2
+
+            res = api_client.get("/api/v3/recipe/?uses_only_baseline_capabilities=true")
+            assert res.status_code == 200
+            assert res.data["count"] == 1
```
Comment (leplatrem) [inline normandy/recipes/tests/api/v3/test_api.py]: I don't obtain the expected result here. Anyone has an idea why?

File: normandy/recipes/tests/api/v3/test_api.py [hunk 2]
```diff
@@ -144,6 +144,21 @@ def test_list_sets_no_cookies(self, api_client):
...
```
Comment (mythmon) [inline normandy/recipes/tests/api/v3/test_api.py]: Can you check that the returned ID is the expected one here?

**Pull request merged**